PHP/git.php

```diff
@@ -22,12 +22,12 @@
 <?php 
 	$count = 0;
 	foreach ($arrayOfNames as $name) {
-			if ($count > 0) {
+			if ($count++ > 0) {
```

Comment:
Agreed. I will increment the counter separate from the comparison, to make code easier to read.